src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -24,26 +24,29 @@ def _complex_params(command_group, engine):
 
         with self.argument_context('{} server create'.format(command_group)) as c:
             c.expand('sku', engine.models.Sku)
-            c.ignore('name', 'family', 'size')
+            c.ignore('size')
```

Comment:
It is intended that these parameters are changed both for postgres and MySQL?

Reply:
yes